client/src/components/train/ModelTable.tsx

```diff
@@ -4,7 +4,14 @@ import {deleteModel, downloadModel, trainModel} from 'lib/api/models';
 import urls from 'lib/urls';
 import Link from 'next/link';
 import React from 'react';
-import {XCircle, Eye, Target, Download, Check} from 'react-feather';
+import {
+  XCircle,
```

Comment:
https://feathericons.com/?query=trash-2 might have less ambiguity

Reply:
Great find, I'm going to make a separate PR updating this across all aspects of the UI